tests/coreos-install/util/util.go

```diff
@@ -28,6 +28,15 @@ import (
 	"testing"
 )
 
+func regexpSearch(name, pattern string, data []byte) (string, error) {
```

Comment:
Let's not have a `regexpSearch` and a `RegexpSearch` with different error semantics.  Please use a descriptive name, and maybe have one call the other.